package.json

```
@@ -3,7 +3,7 @@
   "version": "3.4.0",
   "description": "Current context for LoopBack applications, based on cls-hooked",
   "engines": {
-    "node": "^8.2.1 || ^10.14"
+    "node": ">=8"
```

@ahmetcetin please note that we are intentionally limiting the Node.js versions supported by this module. The implementation depends on Node.js functionality that used to change between major versions of Node, we need to test the module against each new major version before we can claim it as being supported.

I am proposing the following version spec:

```suggestion
    "node": "^8.2.1 || ^10.14 || ^12.15"
```